sdk/metricsadvisor/ai-metrics-advisor/swagger/README.md

```diff
@@ -17,11 +17,10 @@ source-code-folder-path: ./src/generated
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/08f5e391f2153a99580b458cc71ef88e45dd0531/specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json
 add-credentials: false
 override-client-name: GeneratedClient
-use-extension:
```

Comment:
I believe since you removed this `use-extension` & the latest version is installed in your local machine, you are able to generate it without any issues. 

But, the best practice is to mention the version that you use. This will help us avoid confusion in the future. Please update with the version you used similar to https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/search/search-documents/swagger/Data.md#configuration.

Reply:
done